PlatformIO/src/device.h

```diff
@@ -12,7 +12,8 @@ enum {
   COLORS_IDLE = 3,
   COLORS_OTA = 4
 };
-enum {
+enum DeviceMode {
```

Comment:
I have not encountered this problem, but good find :)

Reply:
Unless you have an Audiokit or M5 Atom Echo, you would not be affected. And you have to get the device to reboot (or under some other unlikely circumstances this can also happen when starting from scratch), this issue would not show itself. During my experiments with the RingBuffer I was able reproduce this when I by accident block Core 0 entirely and the task watchdog decided it is time to restart the device (which "ended"  in an endless restart loop.

Reply:
> Unless you have an Audiokit or M5 Atom Echo, you would not be affected. And you have to get the device to reboot (or under some other unlikely circumstances this can also happen when starting from scratch), this issue would not show itself. During my experiments with the RingBuffer I was able reproduce this when I by accident block Core 0 entirely and the task watchdog decided it is time to restart the device (which "ended" in an endless restart loop.

I have a M5 Echo (and a Matrix Voice), but good that it is fixed anyway :)